src/compiler.ts

```diff
@@ -9420,8 +9420,8 @@ export class Compiler extends DiagnosticEmitter {
         let numParameters = parameterTypes.length;
         let operands = new Array<ExpressionRef>(1 + numParameters);
         operands[0] = module.local_get(0, nativeSizeType);
-        for (let i = 0; i < numParameters; ++i) {
```

Comment:
Sorry forget mention that changes also required change exit condition from `i < numParameters` 
to `i <= numParameters`